toolvoice/scripts/inference/cascade_asr/enrich_asr_csvs.py

```
@@ -0,0 +1,74 @@
+W2C_CSV = os.path.join(ASR_DIR, "GPT-4o-Mini-STT-When2Call-GPT.csv")
+
+
+def enrich_confetti() -> None:
+    bfcl = pd.read_csv(BFCL_SRC)
+    asr = pd.read_csv(CONFETTI_CSV)
+
+    idx = asr["source_audio_index"].astype(int)
+    if idx.max() >= len(bfcl):
+        raise RuntimeError(
+            f"source_audio_index out of range: max={idx.max()} but BFCL has {len(bfcl)} rows"
+        )
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)